_63C/fal2.md

```diff
@@ -0,0 +1,12 @@
+---
+question: C10
+title: Do I need to use FAL2 for backchannel assertions?
+---
+
+If identity attributes/PII are sent over the back channel, either in the assertion or in a separate request, then FAL2 is not required in order for the information to be sufficiently protected as stated in the implementation guidelines. FAL1 should provide sufficient protection in this instance because the PII is not exposed to untrusted third parties, such as the browser. This requirement can be found in [Section 4 of SP 800-63C](https://pages.nist.gov/800-63-3/sp800-63c.html#fal).
```

Comment:
What's found there is a conditional requirement to use FAL2. It might be good to state it that way in case they're looking in Section 4 specifically for a statement that it's OK to use FAL1 in the back channel.